packages/story-editor/src/app/rightClickMenu/provider.js

```diff
@@ -41,13 +41,14 @@ import { useStory } from '..';
 import { createPage, duplicatePage, ELEMENT_TYPES } from '../../elements';
```

Comment:
@barklund FYI when starting to review this -- it has a lot of changed files but most of this is cleaning up after removing the Saved Styles panel as it was before so it's not that much really 😛 

About the new `StyleManager` component -- it does have some similarities with the ColorPicker but didn't seem enough to try to create a new component for it, seemed more trouble than benefit.

Reply:
I don't mind this - they're significantly different that refactoring would be more work than gain. And I actually recommend removing the whole edit mode (in a different comment), which would make them even more different.